scripts/ci/run-vpp-tests.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -v
+set -e
+
+dir="$(dirname "$0")"
+TEMP_DIR=${TEMP_DIR:-/tmp/skydive-vpp}
+mkdir -p $TEMP_DIR/skydive-etcd
+
+vpp_create_loopback() {
+    loop=$(vppctl loopback create-interface | tr -d '\r')
+    ret=$?
+    if [ -z $loop ] ; then
+        echo "can't create loopback $loop interface : $ret"
+        exit 1
+    fi
+    # skydive vpp probe poll every 200ms
+    sleep 0.2
+    echo "$loop"
+}
+
+cd ${GOPATH}/src/github.com/skydive-project/skydive
+make WITH_VPP=true VERBOSE=true install
+
+cat <<EOF >> $TEMP_DIR/skydive.yml
+logging:
+  level: DEBUG
+etcd:
+  embedded: true
+  data_dir: $TEMP_DIR/skydive-etcd
+agent:
+  topology:
+    probes:
+      - vpp
+analyzer:
+  flow:
+    backend: elasticsearch
+  topology:
+    backend: elasticsearch
+EOF
+
+echo "VPP tests ..."
+
+loop1=$(vpp_create_loopback)
+
+export SKYDIVE_ANALYZERS="127.0.0.1:8082"
+sudo $GOPATH/bin/skydive allinone -c $TEMP_DIR/skydive.yml &
+SKYDIVE_PID=$!
+sleep 5
+
+t1=$(skydive client query "g.V().Has('Driver','vpp','Name','$loop1')" | wc -l)
+if [ $t1 -le 1 ] ; then
+    echo "$loop1 test failed : $t1"
+    exit 1
+fi
+echo "Test t1 : OK"
```

Comment:
the above block is duplicated 3 times (lines 51-56, 60-65,  69-74) please use a function

Reply:
there are not exactly the same
making a function will not enhance the visibility